server/src/main/java/org/apache/druid/query/lookup/LookupListeningAnnouncerConfig.java

```diff
@@ -57,4 +59,9 @@ public String getLookupTier()
         lookupTierIsDatasource ? "bound value" : LookupModule.PROPERTY_BASE
     );
   }
+
+  public List<String> getLookupsToLoad()
+  {
+    return dataSourceTaskIdHolder.getLookupsToLoad();
```

Comment:
Why do this via `LookupListeningAnnouncerConfig`?
We could just inject the `DatasourceTaskIdHolder` into `LookupReferencesManager`.

Reply:
@kfaraz `LookupListeningAnnouncerConfig` is already injected in `LookupReferencesManager`, and already being used to fetch lookup tier: `lookupListeningAnnouncerConfig.getLookupTier()`, so I added it there itself.

Reply:
Yeah, I guess it's fine to keep it there.